README.md

```diff
@@ -4,103 +4,99 @@ This package provides a set of language helpers, utility functions, and classes
 It includes APIs for feature detection, asynchronous and streaming operations, basic event handling,
 and making HTTP requests.
 
-## Features
+## Installation
 
-### Feature Detection
+This package is currently in Alpha with a initial stable release scheduled for later this year. You can install the Alpha
+through npm:
```

Comment:
I am pretty certain that at least that command does install a compiled version (although, outdated and probably never to be updated) since that alpha version does point to [@bryanforbes core repo](https://github.com/bryanforbes/core).